addons/product/data/product_demo.xml

```diff
@@ -419,6 +419,19 @@
             <field name="image_1920" type="base64" file="product/static/img/product_product_24-image.jpg"/>
         </record>
 
+        <record id="product_product_26" model="product.product">
+            <field name="name">Low Noise Wind Generator</field>
+            <field name="categ_id" ref="product_category_office"/>
+            <field name="standard_price">500</field>
+            <field name="list_price">375</field>
+            <field name="type">consu</field>
+            <field name="weight">3.00</field>
+            <field name="uom_id" ref="uom.product_uom_unit"/>
+            <field name="description_sale">Low Noise 10.000W 220V Wind Generator</field>
+            <field name="default_code">FURN_12123</field>
+            <field name="image_1920" type="base64" file="product/static/img/product_product_26-image.jpg"/>
+        </record>
```

Comment:
Why in product if only used in account ? why can't you use an existing demo data ?
Also, doesn't seem to go very well with the existing demo data which were office related IMO :thinking: